pkg/pillar/types/contenttreetypes.go

```diff
@@ -157,6 +160,10 @@ func (status ContentTreeStatus) IsContainer() bool {
 
 // ReferenceID get the image reference ID
 func (status ContentTreeStatus) ReferenceID() string {
+
+	if base.IsHVTypeKube() && status.IsContainer() {
```

Comment:
base.IsHVTypeKube opens and reads a file, right?
I don't think we want to do that all the time.
So either pass that as an argument or make it be set in a field in ContentTreeStatus.

Reply:
Yeah I will add to ContentTreeStatus

Reply:
Done